src/Tizen.NUI/src/public/XamlBinding/ContentPage.cs

```diff
@@ -109,7 +109,14 @@ public ContentPage(Window win)
             Root.WidthResizePolicy = ResizePolicyType.FillToParent;
             Root.HeightResizePolicy = ResizePolicyType.FillToParent;
 
-            win.Add(Root);
+            if (win != null)
+            {
+                win.Add(Root);
+            }
+            else
+            {
+                NUIApplication.GetDefaultWindow().Add(Root);
```

Comment:
If the parameter window is null, would not it be better not to act than to attach it to the main window?

Reply:
Accepted your opinion! :)